dotfiles.py

```diff
@@ -1,29 +1,52 @@
 #!/usr/bin/env python3.8
-"""Module docstring."""
+
 import filecmp
+from collections import namedtuple
+from os.path import exists
 from os.path import expanduser
 from shutil import copyfile
 from typing import List
-from typing import Tuple
+from typing import NamedTuple
 
 import click
 
-DOTFILES: List[Tuple[str, str]] = [("~/.tmux.conf", "tmux.conf"), ("~/.vimrc", "vimrc")]
+
+class Dotfile(NamedTuple):
```

Comment:
Since both of the attributes for this class represent paths you could leave the `_path` and `_name` suffixes off and just write
```python
class Dotfile(NamedTuple):
    system: Path
    repo: Path
```

Reply:
I like this change; it declutters the code. But, it also makes the code a little less clear even with the `Path` type annotated. Mostly when using it later on without the type annotations:

```python
for system, repo in DOTFILES:
    copyfile(repo, system)
```
vs.
```python
for system_path, repo_path in DOTFILES:
    copyfile(repo_path, system_path)
```

So I'm not 100% sold on dropping the suffixes. 🤔

Reply:
For me at least, the second example is more "clear" about what the code is doing.

Reply:
That's fair. I think this is mostly a matter of personal opinion and that either approach is fine. Personally, I dislike writing the type of the variable after its name (e.g. `foo_list`, `bar_str`, `baz_int`, etc.), but that's just my style. :sweat_smile:

Reply:
I also find that irritating. But in this case, it isn't just the variable type; it's also literally what the variable represents. `system` & `repo` are pretty generic words, so I could probably just pick better variable names. Ugh, naming things.

Reply:
¯\\\_(ツ)_/¯